README.md

```diff
@@ -110,36 +131,36 @@ Go back to the GitHub App and update the following settings
+| *<a name="github_webhook_secret_key_vault_id"></a> [github\_webhook\_secret\_key\_vault\_id](#github\_webhook\_secret\_key\_vault\_id) | Keyvault Vault ID for GitHub App webhook secret ([data source](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/key_vault_secret)) | `string` |
+| *<a name="github_private_key_key_vault_id"></a> [github\_private\_key\_key\_vault\_id](#github\_private\_key\_key\_vault\_id) | Keyvault Vault ID for GitHub App private key ([data source](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/key_vault_secret)) | `string` |
+| *<a name="owners"></a> [owners](#owners) | The list of owners that will be assigned to all components ([data source](https://registry.terraform.io/providers/hashicorp/azuread/latest/docs/data-sources/users)) | `list(string)` |
 
 ## Optional Inputs
 
 One goal of this module is to minimize the number of customizations needed in order to run autoscaling self-hosted runners.  With this being said, this list of optional inputs will grow but hopefully not so much that it becomes difficult to manage and get started with this solution.
 
 | Name | Description | Type | Default |
 |------|-------------|------|---------|
-| [log_level]() | Log level used across applications | `string` | Information |
-| [azure_gallery_name]() | Azure Compute Gallery to be used in runner creation, leave default to use the Liatrio-provided image | `string` | /subscriptions/3d243cec-9a80-435e-8bcd-4349b654b665/resourceGroups/rg-liatrio-community-gallery |
-| [azure_gallery_image_id]() | Azure Compute Gallery image ID to be used in runner creation, leave default to use `latest` Liatrio public image | `string` | /providers/Microsoft.Compute/galleries/liatrioCommunityGalleryTest/images/ubuntu_gh_runner/versions/0.0.4 |
-| [event_handler_image_tag]() | Event-Handler image tag to use from [GitHub Packages](https://github.com/liatrio/terraform-azure-github-runner/pkgs/container/github-webhook-runner-controller) | `string` | latest |
-| [runner_controller_image_tag]() | Runner-Controller image tag to use from [GitHub Packages](https://github.com/liatrio/terraform-azure-github-runner/pkgs/container/github-webhook-runner-controller) | `string` | latest |
+| <a name="log_level"></a> [log\_level](#log\_level) | Log level used across applications | `string` | Information |
+| <a name="azure_gallery_image_id"></a> [azure\_gallery\_image\_id](#azure\_gallery\_image\_id) | Azure Compute Gallery image ID to be used in runner creation, leave default to use `latest` Liatrio public image | `string` | /communityGalleries/liatrio-4e8ffc8d-5950-4137-b02c-df028384cdcd/images/ubuntu_gh_runner/versions/latest |
```

Comment:
I don't know if there's a great way to fix it, but this line + github's layout makes this table render awkwardly.

Reply:
I've now added `<br />` breaks in the default value string so it forces new lines and allows for the description column to widen. If a user copies the default value from the table it will have newlines copied with it, but I think that inconvenience is outweighed by easier to read documentation.